tests/metrics/collect_mem_consmd.sh

```diff
@@ -128,38 +134,115 @@ function start_collect()
 # It kills the collectl process.
 function stop_collect()
 {
+	echo "Stop collectl"
 	kill "$COLLECT_PID"
 }
 
-# Get the average about a "csv" file produced
-# by collectl tool.
-function get_average()
+# It launches a instance of psstop tool
+# in order to get metrics about proportional
+# set size.
+function start_psstop()
 {
+	echo "Launching psstop"
+	psstop_monitor=$(cat << EOF
+	#!/bin/bash
+
 	sum=0
 	count=0
 
+	while [ 1 ];do
+		if [ ! -f "$STOP_FILE" ];then
+			data=\$(psstop $PROC | awk '{print \$1}')
```

Comment:
`psstop` is packaged for Clear, but is not available in the Ubuntu and Fedora repos. An alternative tool that shows PSS that *is* available in all 3 distros is `smem`. In fact, https://github.com/01org/cc-oci-runtime/blob/master/tests/metrics/density/docker_memory_usage.sh.in is already using that.

What is the overlap between this script and the existing one?